src/components/NavBar/DropDownMenu.tsx

```diff
@@ -0,0 +1,33 @@
+import { faCaretDown, faTools } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import NavBarTools from './NavBarTools';
+import NavBarStyle from './NavBar.module.css';
+import DropDownStyle from '../BasicComponents/DropDownMenu.module.css';
+
+interface MenuProps {
+  platform: {
+    id: string;
+    header: string;
+    style: string;
+  };
+}
+
+const DropDownMenu = ({ platform }: MenuProps) => (
```

Comment:
you have two options
 1. create generic DropDownMenu that get items /item (with id, title(or header as you call it ), style and maybe even optional icon)  and send to it the platform but' you will be able to reuse it

2. chnage the name of the component  to something like 'platformDropDown'